lib/lightning/kafka_triggers.ex

```diff
@@ -33,7 +33,15 @@ defmodule Lightning.KafkaTriggers do
   Selects the appropriate offset reset policy for a given trigger based on the
   presence of partition-specific timestamps.
   """
-  def determine_offset_reset_policy(trigger) do
+  def determine_offset_reset_policy(trigger, opts \\ []) do
+    if rollback_timestamp = opts |> Keyword.get(:rollback_timestamp) do
```

Comment:
@jyeshe Except it is not actually an offset that we are passing in but a timestamp, so at this level I really can't talk about an offset as it may be confusing. Can you live with `reset_timestamp`?

Reply:
@jyeshe I have renamed all cases of `rollback_*` to `reset_*` in a follow-up commit.

Reply:
@rorymckinley thanks, you got it, I meant `reset` when talked about offset.